app/grandchallenge/emails/admin.py

```diff
@@ -0,0 +1,96 @@
+from django.contrib import admin, messages
+from django.db.transaction import on_commit
+from django.utils.timezone import now
+from django.utils.translation import ngettext
+from markdownx.admin import MarkdownxModelAdmin
+
+from grandchallenge.emails.models import Email
+from grandchallenge.emails.tasks import send_bulk_email
+from grandchallenge.emails.utils import SendActionChoices
+
+
+def schedule_emails(modeladmin, queryset, request, action):
+    emails = queryset.filter(sent=False)
+    if emails:
+        for email in emails:
+            send_admin_emails = send_bulk_email.signature(
+                kwargs={
+                    "action": action,
+                    "subject": email.subject,
+                    "body": email.body,

```

Comment:
The task is still taking the email subject and body, which could be a problem with the limited message size of SQS. It would be better if the task takes the primary key to the email, and then fetches the subject and body from the database.